cmd/bdjuno/main.go

```diff
@@ -67,5 +71,18 @@ func getBasicManagers() []module.BasicManager {
 func getAddressesParser() messages.MessageAddressesParser {
 	return messages.JoinMessageParsers(
 		messages.CosmosMessageAddressesParser,
+		OsmoMessageAddressesParser,
```

Comment:
Since these are called in the same order in which they are specified, they should be inverted: 
```suggestion
		OsmoMessageAddressesParser,
		messages.CosmosMessageAddressesParser,
```
This way even if the custom `OsmoMessageAddressesParser` doesn't take care of a message, the `CosmosMessageAddressesParser` can do it instead